src/main/webapp/WEB-INF/views/users/login.jsp

```diff
@@ -0,0 +1,20 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Login</title>
+</head>
+<body>
+<h1>Login page</h1>
+<h4 style="color: crimson">${errorMessage}</h4>
+<form method="post" action="${pageContext.request.contextPath}/login">
+    <label for="login">Login:</label><br>
+    <input type="text" id="login" name="login"><br>
+    <label for="pwd">Password:</label><br>
+    <input type="password" id="pwd" name="pwd"><br>
+    <input type="submit" value="Login">
+</form>
+<br>
+<form action="${pageContext.request.contextPath}/registration">
+    <input type="submit" value="Registration!"><br></form>
```

Comment:
you had better use "Login!" instead of "Registration!"

Reply:
I not clear what you mean.
There is exist button "Login!". "Registrations" is another button.